Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -324,6 +324,9 @@ ErrorOr<NonnullRefPtr<WebDriverConnection>> WebDriverConnection::connect(Web::Pa
     dbgln_if(WEBDRIVER_DEBUG, "Trying to connect to {}", webdriver_ipc_path);
     auto socket = TRY(Core::LocalSocket::connect(webdriver_ipc_path));
 
+    // Allow pop-ups, or otherwise /window/new won't be able to open a new tab.
+    page_client.page().set_should_block_pop_ups(false);
```

Comment:
Can you explain this more in the commit message? Why does blocking popups prevent new tabs from opening?

Reply:
sure, done